### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by egikyan**
- Add simple Dockerfile
- Change access of external repo as github deprecated [unsecure git](https://github.blog/2021-09-01-improving-git-protocol-security-github/#no-more-unauthenticated-git)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# Meta
+LABEL Description="Stremio Web" Vendor="Smart Code ltd" Version="1.0.0"
+
+# Create app directory
+RUN mkdir -p /var/www/stremio-web
+
+# Install app dependencies
+WORKDIR /var/www/stremio-web
+COPY package.json package-lock.json /var/www/stremio-web
```
Comment (core1024) [inline Dockerfile]: This step doesn't work, at least with my version of Docker

```
Step 5/12 : COPY package.json package-lock.json /var/www/stremio-web
When using COPY with more than one source file, the destination must be a directory and end with a /
```
 Also it is redundant as latter on you do `COPY . /var/www/stremio-web` that copies all the files. You can just move that line here.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+RUN npm install -g http-server
+
+# Bundle app source
+WORKDIR /var/www/stremio-web
+COPY . /var/www/stremio-web
+
+RUN npm run build 
+
+EXPOSE 8080
+CMD ["http-server", "/var/www/stremio-web/build/", "-p", "8080"]
```
Comment (core1024) [inline Dockerfile]: Would you also add `-d false` here. Disabling directory listings is always a good idea.

**Review by core1024 — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Stremio Node 14.x
+FROM stremio/node-base:fermium
+
+# Meta
+LABEL Description="Stremio Web" Vendor="Smart Code ltd" Version="1.0.0"
```
Comment (nklhtv) [inline Dockerfile]: should be "Smart Code OOD"

File: Dockerfile (hunk 1 above)
Comment (egikyan) [reply Dockerfile]: Done

File: Dockerfile (hunk 2 above)
Comment (egikyan) [reply Dockerfile]: Done

File: Dockerfile (hunk 3 above)
Comment (egikyan) [reply Dockerfile]: Done

**Review by core1024 — APPROVED** [review]

**Pull request merged**